ClearKeep/Scenes/Home/Models/CallViewModel.swift

```diff
@@ -0,0 +1,454 @@
+//
+//  CallViewModel.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 19/07/2022.
+//
+
+import Foundation
+import WebRTC
+import ChatSecure
+import ChatSecure
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Models/CallViewModel.swift#L11"></span>Imports should be unique.<br />
<code>duplicate_imports</code> <code>CallViewModel.swift:11</code></td>
    </tr>
  </tbody>
</table>